lib/icinga/externalcommandprocessor.cpp

```diff
@@ -1101,11 +1113,18 @@ void ExternalCommandProcessor::DelDowntimeByHostName(double, const std::vector<S
 		Log(LogWarning, "ExternalCommandProcessor")
 			<< ("Ignoring additional parameters for host '" + arguments[0] + "' downtime deletion.");
 
+	String downTimeName;
```

Comment:
I'd declare this within the loop and use a lowercase `t` to match the capitalization of the other occurrances of downtime.